app/components/card/care-card-emergency.njk

```diff
@@ -22,6 +24,7 @@
           <p>You could be having a heart attack. Call 999 immediately as you need immediate treatment in hospital.</p>
```

Comment:
should this now say
```suggestion
          <p>You could be having a heart attack. Call 999 immediately as you need emergency treatment in hospital.</p>
```

Reply:
Good spot, but I think, for the public, leave it as it is, @DomBaker.